openapi/components/schemas/RiskScore/RiskScoreInteger.yaml

```
@@ -0,0 +1,12 @@
+type: object
+nullable: true
+required:
+  - scoreValue
+  - threshold
+properties:
+  scoreValue:
+    type: integer
+    description: This value is added to transaction risk score.
+  threshold:
+    type: integer
+    description: Condition that has to be reached by a risk factor to increase transaction risk score.
```

nitpick: I found this one difficult to interpret (I made assumptions here)
```suggestion
    description: |-
      Threshold of the risk score.
      If this value is exceeded by a risk factor, the risk score of the related transaction is increased.
```